catalog/transformer/catalog-transformer-tika-input/src/main/java/ddf/catalog/transformer/input/tika/TikaInputTransformer.java

```diff
@@ -506,14 +506,10 @@ protected MetacardType mergeAttributes(MetacardType metacardType) {
             .flatMap(Collection::stream)
             .collect(Collectors.toSet());
 
-    // Guard against empty collection. If the collection is empty,
-    // the MetacardTypeImpl constructor throws an exception.
-    if (!additionalAttributes.isEmpty()) {
-      returnObject =
-          new MetacardTypeImpl(metacardType.getName(), metacardType, additionalAttributes);
-    }
+    // always add extracted attributes
+    additionalAttributes.addAll(new ExtractedAttributes().getAttributeDescriptors());
```

Comment:
If the ExtractedAttributes descriptors are being added here should they be removed from the blueprint file?

Here, for example:
https://github.com/codice/ddf/blob/master/catalog/transformer/catalog-transformer-tika-input/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L63